Shared/Models/LoopringV3.cs

```diff
@@ -496,4 +496,15 @@ public class PairWeeklyData : PairEntity
         public double weekNumber { get; set; }
     }
 
+    public class NftHolder
+    {
+        public MinimalAccount? Account { get; set; }
+        public int Balance { get; set; }
+    }
+
+    public class MinimalAccount
+    {
+        public string? Id { get; set; }
+        public string? Address { get; set; }
+    }
```

Comment:
Why not reuse the Account class? It is quite normal for it to be only partially filled in the various queries.

Reply:
Actually our model in `LoopringV3.cs` tries to match the graph scheme, omitting properties we're not using. Some early mistakes there caused some diversion, but that's the general idea.

So consequently even the `NftHolder` class is not necessary and it should actually use the existing AccountNFTSlot.

Reply:
Hmm... I'm a bit against this in general. This means we could have one giant class with all properties, even though the properties are never used or set. Specific cases should really have specific classes. 

There's also alot of nullable properties and I guess this is why. 

If there's a specific reason for doing so, cause things break or something I am not aware of, I'll do it. But after rethinking this also about the Account I really think we should use specific DTOs.

Reply:
1. This is the way we've done it up until now. Either we always stick to the rule or never.
2. It is perfectly fine to fill up the DTO partially with several queries. Like it is done for account balances. First we fetch the account, then we fetch the balances and add them to the already existing account.
3. Every line of code is a liability that has to be managed in the future. Now adding dozens of classes which all describe the same entity but with varying properties for every use-case looks like a maintenance nightmare to me.
4. This would also break the JsonSubTypes concept - how will it decide when looking for a class that matches a typename?

Reply:
Fixed :)